### PR TITLE
Release Note for 1.6.0

### PR DESCRIPTION
**Pull request by chunseoklee**
- Release Note for 1.6.0

ONE-DCO-1.0-Signed-off-by: Chunseok Lee <chunseok.lee@samsung.com>

### CODE REVIEW

File: docs/release/1.6/release-note-1.6.0.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+### CPU backend supports more operations
+
+- BatchMatMul, BroadcastTo, Einsum, FusedBatchNorm, MatrixBandPart, Range,  ReduceAll, Add(quant8)
+
+### Support dynamically shaped tensors
+
+- Support static shape inference (input resizing)
+- Support dynamic shape inference (general resizing)
+### Support Control Flow operations
```
Comment (seanshpark) [inline docs/release/1.6/release-note-1.6.0.md:32]: Add an empty line above this ?

**Comment by chunseoklee** [comment]
@Samsung/nnfw 
PTAL

File: docs/release/1.6/release-note-1.6.0.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+### Support Control Flow operations
+
+- IF and WHILE
+- Fully support static and dynamic tensors
+
+### API updates
+
+- Introduce `nnfw_set_input_tensorinfo` for input resizing
+- `nnfw_input_tensorinfo` and `nnfw_output_tensorinfo` behavior have changed to return updated tensorinfo
```
Comment (wateret) [inline docs/release/1.6/release-note-1.6.0.md:42]:
```suggestion
- `nnfw_input_tensorinfo` and `nnfw_output_tensorinfo` behavior have changed to return tensorinfo according to the session state
```

File: docs/release/1.6/release-note-1.6.0.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
+# Release Note 1.6.0
+
+## Feature Highlights
+
+- **ONE** Compiler
+    - Compiler supports more operations
```
Comment (lemmaa) [inline docs/release/1.6/release-note-1.6.0.md:6]: How about more detailed information?

```suggestion
    - Compiler supports 22 more operations including 1 custom operation.
```

File: docs/release/1.6/release-note-1.6.0.md [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+
+- **ONE** Runtime
+    - CPU backend supports more operations
```
Comment (lemmaa) [inline docs/release/1.6/release-note-1.6.0.md:9]:
```suggestion
    - CPU backend supports 8 more operations
```

**Comment by chunseoklee** [comment]
suggestions applied.

File: docs/release/1.6/release-note-1.6.0.md [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+
+- **ONE** Compiler
+    - Compiler supports 22 more operations including 1 custom operation.
+
+- **ONE** Runtime
+    - CPU backend supports 8 more operations
+    - Support dynamically shaped tensors
+    - Support Control Flow operations
+    - API updates
+
```
Comment (lemmaa) [inline docs/release/1.6/release-note-1.6.0.md:13]:
```suggestion

- Add **ONE** Runtime package for `x86_64` linux (Experimental)

```

**Pull request merged**